File: samples/snippets/cpp/VS_Snippets_CLR/Environment.NewLine/CPP/newline.cpp
//<snippet1>
// Sample for the Environment::NewLine property
using namespace System;

int main()
{
   Console::WriteLine();
   Console::WriteLine("NewLine: {0}  first line {0}  second line", Environment::NewLine);
}

/*
This example produces the following results:

NewLine:
first line
second line
*/
//</snippet1>

File: samples/snippets/visualbasic/VS_Snippets_CLR/Environment.NewLine/VB/newline.vb
'<snippet1>
' Sample for the Environment.NewLine property
Class Sample
   Public Shared Sub Main()
      Console.WriteLine()
      Console.WriteLine($"NewLine:{Environment.NewLine}  first line{Environment.NewLine}  second line")
   End Sub
End Class

'This example produces the following results:
'
'NewLine:
'  first line
'  second line
'
'</snippet1>
